pkg/collector/corechecks/cluster/kube-state-metrics_test.go

```diff
@@ -639,3 +732,13 @@ func TestKSMCheck_joinLabels(t *testing.T) {
 		})
 	}
 }
+
+func lenMetrics(metricsToProcess map[string][]ksmstore.DDMetricsFam) int {
+	var count int
```

Comment:
```suggestion
	count := 0
```
would make the initialisation more explicit.